docs/user/05-60-kyma-bindings.md

```diff
@@ -40,21 +40,23 @@ Now, the unmarshalled request is validated, and the correctness of its structure
 
 After unmarshaling, the data is validated against the allowed parameter values, which includes checks of the expiration value range, database record existence, parameter mutations verification, expiration of existing bindings, and binding number limits. 
 The first check verifies the expiration value. The minimum and maximum limits are configurable and, by default, set to 600 and 7200 seconds, respectively. 
-Next, KEB checks if the binding already exists. The binding in the database is identified by the Kyma instance ID and the binding ID, which are passed as a path query parameter. If the binding exists, KEB checks the mutation of the parameters of the existing binding. 
+Next, KEB checks the status of the instance. The instance must be provisioned for the binding creation. 
+After checking the instance, KEB checks if the binding already exists. The binding in the database is identified by the Kyma instance ID and the binding ID, which are passed as a path query parameter. If the binding exists, KEB checks the mutation of the parameters of the existing binding. 
```

Comment:
On the diagram if the service binding (the pair `{service binding, service instance}`) already exists but with the different parameters we continue. The logic is reversed.